spec/features/floof_creation_spec.rb

```diff
@@ -0,0 +1,30 @@
+
+require 'rails_helper'
+
+feature 'Migration Index' do
```

Comment:
this does not really test anything from a user facing feature standpoint. It would be good to supply model tests for the `Floof` model